operations/UAS_user_and_admin_topics/Troubleshoot_Broker_SSSD_Cant_Use_sssd_conf.md

```diff
@@ -0,0 +1,151 @@
+[Top: User Access Service (UAS)](User_Access_Service_UAS.md)
+
+[Next Topic: Clear UAS Configuration](Reset_the_UAS_Configuration_to_Original_Installed_Settings.md)
+
+## Troubleshoot Broker UAI SSSD Can't Use `/etc/sssd/sssd.conf`
+
+### Symptom
+
+A Broker UAI has been created using an SSSD configuration in a secret and volume as described in [Configure a Broker UAI Class](Configure_a_Broker_UAI_Class.md) but logging into the broker does not work.  Upon investigation using
```

Comment:
- Add a comma after [Configure a Broker UAI Class](Configure_a_Broker_UAI_Class.md)
- Capitalize "broker" for consistency
- Remove "Upon investigation using"

Reply:
Done.